tests/data/permutations/all_roman_numbers.csv

```diff
@@ -0,0 +1,3999 @@
+number,text
+1,I
+2,II
+3,III
+4,IV
```

Comment:
According to [this](https://blog.prepscholar.com/roman-numerals-converter#:~:text=*Though%20normally%20considered%20incorrect%20usage,numerals%20is%20MMMCMXCIX%2C%20or%203%2C999.), while technically incorrect, there’s cases were incorrect numbers like `IIII` instead of `IV` is used. Maybe we could support that as well.